### DISCUSSION TITLE
How to delete sidecars?

### DISCUSSION THREAD

**Discussion by ssingularity** [argoproj/argo-workflows#13573] [Q&A] [answered]
Could anyone tell me What is the current way to delete sidecars in Argoworkflow?
It seems that the #8475 has remove the killSideCars method and depreacted the kill operation in wait container

**Comment by agilgur5** [comment, marked as answer]
Do you mean killing _injected_ sidecars? If so, there is a docs page on it: https://argo-workflows.readthedocs.io/en/release-3.5/sidecar-injection/

If you mean `sidecars` defined in the spec, they are containers in the Pod and so follow the Pod lifecycle
